cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -138,7 +141,23 @@ Tables. These Tables are similar to tables in a relational database with a few k
   and written independently of other columns, and columns are ordered
   in byte-lexicographic order. They are also known as *Ordered Columnar Tables*.
 
-|fileset|_ is a core Dataset type that was added as an experimental feature in CDAP 2.6.0.
+
+A FileSet represents a collections of files in the file system that share some common attributes
```

Comment:
This will bring back the link, as the bold text "FileSet":
```
A |fileset|_ represents a collections of files in the file system that share some common attributes
```